spectrum/spectrum/settings.py

```diff
@@ -120,7 +120,7 @@
 
 USE_L10N = True
 
-USE_TZ = True
+# USE_TZ = True
```

Comment:
If we're not going to use time zones, should we just delete the line (rather than commenting it out)?